test-suite/src/inline_view.rs

```diff
@@ -0,0 +1,253 @@
+use {
```

Comment:
Test cases looks beautiful, easily readable, very informative and well aligned 👍